Source/WebKit/UIProcess/Notifications/WebNotificationManagerProxy.h

```diff
@@ -28,6 +28,7 @@
 #include "APIObject.h"
 #include "MessageReceiver.h"
 #include "WebContextSupplement.h"
+#include "WebNotificationIdentifier.h"
```

Comment:
Can this header use a forward declaration instead?

Reply:
Yes. Will do.